api/v1beta1/packetcluster_webhook.go

```diff
@@ -47,6 +47,23 @@ func (c *PacketCluster) Default() {
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type.
 func (c *PacketCluster) ValidateCreate() error {
 	clusterlog.Info("validate create", "name", c.Name)
+	allErrs := field.ErrorList{}
```

Comment:
I think we could update these validations to be more explicit that `metro` is the way forward, and `facility` is supported temporarily for backwards compatibility.